pkg/apis/fleet/v1alpha1/types.go

```diff
@@ -394,26 +394,23 @@ type MonSpec struct {
 	// +optional
 	Count *int `json:"count,omitempty"`
 
-	// In a ceph cluster, it is recommended that the monitor pod be deployed on a different node in order to ensure high availability of data.
-	// In practice, you can label the node where the monitor pod is deployed with Annotation/Labels.
-	// Then use kubernetes node affinity rules to achieve the goal of deploying the monitor to different nodes.
-	// The annotations-related configuration to add/set on each Pod related object.
+	// The annotation-related configuration to add/set on each Pod related object. Including Pod， Deployment.
 	// +nullable
 	// +optional
 	Annotations rookv1.AnnotationsSpec `json:"annotations,omitempty"`
 
 	// Similar to Annotation, but more graphical than Annotation.
-	// The labels-related configuration to add/set on each Pod related object.
+	// The label-related configuration to add/set on each Pod related object. Including Pod， Deployment.
 	// +kubebuilder:pruning:PreserveUnknownFields
 	// +nullable
 	// +optional
-	Labels rookv1.LabelsSpec `json:"labels,omitempty"`
+	Labels map[string]string `json:"labels,omitempty"`
 
 	// The placement-related configuration to pass to kubernetes (affinity, node selector, tolerations).
 	// +kubebuilder:pruning:PreserveUnknownFields
 	// +nullable
 	// +optional
-	Placement rookv1.PlacementSpec `json:"placement,omitempty"`
+	Placement map[string]string `json:"placement,omitempty"`
```

Comment:
This is not right, 

we need such struct
https://github.com/rook/rook/blob/ffaf4a1b3fcea3e0cabb6436328f643ad59407f8/pkg/apis/ceph.rook.io/v1/types.go#L2763C24-L2779